### PR TITLE
Specify scope while update progress

### PR DESCRIPTION
**Pull request by live1206**
[AB#344730](https://ceapex.visualstudio.com/Engineering/_workitems/edit/344730)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/6848)

### CODE REVIEW

File: src/docfx/build/toc/TocMap.cs [hunk 1]
```diff
@@ -159,21 +159,21 @@ private static (int subDirectoryCount, int parentDirectoryCount) GetRelativeDire
         private (FilePath[] tocs, Dictionary<FilePath, FilePath[]> docToTocs, List<FilePath> servicePages)
             BuildTocMap()
         {
-            using (Progress.Start("Loading TOC"))
+            using (var scope = Progress.Start("Loading TOC"))
```
Comment (yufeih) [inline src/docfx/build/toc/TocMap.cs:162]: remove `{` and `}` if we are measuring the function?
```suggestion
            using var scope = Progress.Start("Loading TOC");
```

File: src/docfx/lib/log/Progress.cs [hunk 2]
```diff
@@ -8,17 +8,13 @@
 
 namespace Microsoft.Docs.Build
 {
-    internal static class Progress
+    internal static partial class Progress
```
Comment (yufeih) [inline src/docfx/lib/log/Progress.cs:11]: nit: why partial
```suggestion
    internal static class Progress
```

**Review by yufeih — APPROVED** [review]
👍

File: src/docfx/lib/log/Progress.cs (hunk 2 above)
Comment (live1206) [reply src/docfx/lib/log/Progress.cs:11]: Mistakenly added by VS while moving `LogScope` out...

**Pull request merged**